test/inductor/test_triton_kernels.py

```diff
@@ -3504,9 +3505,18 @@ def grid(META):
         w = torch.randn(K, N, device=GPU_TYPE)
 
         torch._dynamo.decorators.mark_unbacked(x, 0)
-        torch._logging.set_logs(output_code=True)
-        with self.assertLogs(logger="torch._inductor", level=logging.DEBUG) as log:
-            foo(x, w)
+
+        @contextlib.contextmanager
```

Comment:
Nit should be an enable output_code_logs() context manager that restore the original state.

Reply:
thanks for the review @Skylion007 !

quick question about this nit - did you mean that this context manager should restore the original state instead of restoring to `output_logs=False` ?

Reply:
yep, just in case. good code design.

Reply:
I agree this would be ideal, but I don't see an easy way to obtain the original state without making some more changes in `torch._logging`

Reply:
thanks @SamGinzburg for the suggestion to use `logging_utils`, which has a context manager for patching and restoring logging settings